app/models/application_key.rb

```diff
@@ -2,6 +2,7 @@ class ApplicationKey < ApplicationRecord
 
   KEYS_LIMIT = 5
 
+  audited
```

Comment:
Maybe we do not want to audit all the fields

Reply:
which ones we do not want to? check the test to see the ones I am auditing :smile: I guess it is reasonable but I am opened to suggestions.

Reply:
I think we should whitelist them. Example it does not make sense to audit `tenant_id`

Reply:
Makes sense :+1: :smile: